cli/tests/unit/kv_queue_test.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { assertRejects } from "./test_util.ts";
+
+Deno.test({}, async function queueTestDbClose() {
+  const db: Deno.Kv = await Deno.openKv(":memory:");
+  db.close();
+  await assertRejects(
+    async () => {
+      await db.listenQueue(() => {});
```

Comment:
normally `listenQueue` is called without `await`. can we test here that it throws without await?

Reply:
Locally, it's failing without `await`. `listenQueue` return promise.

Reply:
Fixed, I handled using try-catch :).

Reply:
My test cases are passing, but something else is failing @igorzi can you please check?